### PR TITLE
feat: Option on item's page to add/remove from watchlist

### LINKED ISSUE #730: Add ability to add item to Watchlist from actual item page
### Description

Right now it seem only possible way to add item to your watchlist in Jellyseer is to do that from search page by clicking star button present over item poster.
I usually tend to go inside the item to check the ratings and decide whether it worth to be added to Watchlist.
But inside item page I dont see a way to add it to Watchlist and I need to go back and add it by searching it again.
It would be great to have this functionality also inside item page. Maybe the same star button over item cover or separate button/

### Desired Behavior

Have ability to add item to your Jellyseer Watchlist from actual item page.

### Additional Context

_No response_

### Code of Conduct

- [X] I agree to follow Jellyseerr's Code of Conduct

### PR DESCRIPTION
**Pull request by JoaquinOlivero**
re #730

#### Description
Adds a button on the individual media item's page to add or remove it from the user's watchlist.

#### Screenshot (if UI-related)
output

#### To-Dos

- [x] Successful build `yarn build`
- [x] Translation keys `yarn i18n:extract`
- [ ] Database migration (if required)

#### Issues Fixed or Closed

- Fixes #730

### CODE REVIEW

File: src/i18n/locale/en.json [hunk 1]
```diff
@@ -978,7 +983,7 @@
   "components.Settings.plexlibraries": "Plex Libraries",
   "components.Settings.plexlibrariesDescription": "The libraries Jellyseerr scans for titles. Set up and save your Plex connection settings, then click the button below if no libraries are listed.",
   "components.Settings.plexsettings": "Plex Settings",
-  "components.Settings.plexsettingsDescription": "Configure the settings for your Plex server. Jellyseerr scans your Plex libraries to determine content availability.",
```
Comment (fallenbagel) [inline src/i18n/locale/en.json]: This should not be changed

File: src/components/MovieDetails/index.tsx [hunk 2]
```diff
@@ -93,6 +99,13 @@ const messages = defineMessages({
   rtaudiencescore: 'Rotten Tomatoes Audience Score',
   tmdbuserscore: 'TMDB User Score',
   imdbuserscore: 'IMDB User Score',
+  watchlistSuccess:
+    '<strong>{title}</strong> added to watchlist  successfully!',
```
Comment (fallenbagel) [inline src/components/MovieDetails/index.tsx]: Little nitpick but remove the extra whitespace after `watchlist`

File: src/components/MovieDetails/index.tsx [hunk 3]
```diff
@@ -93,6 +99,13 @@ const messages = defineMessages({
...
+  watchlistDeleted:
+    '<strong>{title}</strong> Removed from watchlist  successfully!',
```
Comment (fallenbagel) [inline src/components/MovieDetails/index.tsx]: Here as well

File: src/components/TvDetails/index.tsx [hunk 4]
```diff
@@ -88,6 +100,13 @@ const messages = defineMessages({
   rtcriticsscore: 'Rotten Tomatoes Tomatometer',
   rtaudiencescore: 'Rotten Tomatoes Audience Score',
   tmdbuserscore: 'TMDB User Score',
+  watchlistSuccess:
+    '<strong>{title}</strong> added to watchlist  successfully!',
```
Comment (fallenbagel) [inline src/components/TvDetails/index.tsx]: Refer to my previous comment

File: src/components/TvDetails/index.tsx [hunk 5]
```diff
@@ -88,6 +100,13 @@ const messages = defineMessages({
...
+  watchlistDeleted:
+    '<strong>{title}</strong> Removed from watchlist  successfully!',
```
Comment (fallenbagel) [inline src/components/TvDetails/index.tsx]: Refer to my previous comment

File: src/i18n/locale/en.json [hunk 6]
```diff
@@ -311,6 +313,9 @@
   "components.MovieDetails.theatricalrelease": "Theatrical Release",
   "components.MovieDetails.tmdbuserscore": "TMDB User Score",
   "components.MovieDetails.viewfullcrew": "View Full Crew",
+  "components.MovieDetails.watchlistDeleted": "<strong>{title}</strong> Removed from watchlist  successfully!",
```
Comment (fallenbagel) [inline src/i18n/locale/en.json]: Refer to my previous comment (extraction after fixing my previous comment should fix this)

File: src/i18n/locale/en.json [hunk 7]
```diff
@@ -311,6 +313,9 @@
...
+  "components.MovieDetails.watchlistError": "Something went wrong try again.",
+  "components.MovieDetails.watchlistSuccess": "<strong>{title}</strong> added to watchlist  successfully!",
```
Comment (fallenbagel) [inline src/i18n/locale/en.json]: Refer to my previous comment

File: src/i18n/locale/en.json [hunk 8]
```diff
@@ -1091,6 +1098,9 @@
   "components.TvDetails.streamingproviders": "Currently Streaming On",
   "components.TvDetails.tmdbuserscore": "TMDB User Score",
   "components.TvDetails.viewfullcrew": "View Full Crew",
+  "components.TvDetails.watchlistDeleted": "<strong>{title}</strong> Removed from watchlist  successfully!",
```
Comment (fallenbagel) [inline src/i18n/locale/en.json]: Refer to my previous comment

File: src/i18n/locale/en.json [hunk 9]
```diff
@@ -1091,6 +1098,9 @@
...
+  "components.TvDetails.watchlistError": "Something went wrong try again.",
+  "components.TvDetails.watchlistSuccess": "<strong>{title}</strong> added to watchlist  successfully!",
```
Comment (fallenbagel) [inline src/i18n/locale/en.json]: Refer to my previous comment

**Review by Fallenbagel — CHANGES_REQUESTED** [review]

**Review by gauthier-th — APPROVED** [review]
LGTM!
Thanks for this nice feature :)

**Review by Fallenbagel — APPROVED** [review]

**Comment by Gauvino** [comment]
Could I just run the PR myself to see if everything ok before merging?

**Comment by gauthier-th** [comment]
> Could I just run the PR myself to see if everything ok before merging?

Go ahead

**Review by gauthier-th — APPROVED** [review]

**Pull request merged**